probe_scraper/runner.py

```diff
@@ -62,21 +94,13 @@ def main(temp_dir, out_dir):
                 results = PARSERS[probe_type].parse(paths, details["version"])
                 probes[channel][node_id][probe_type] = results
 
-    # Transform extracted data.
+    # Transform extracted data: get both the monolithic and by channel probe data.
     revisions = transform_revisions.transform(node_data)
-    probe_data = transform_probes.transform(probes, node_data)
+    probes_by_channel = transform_probes.transform(probes, node_data, True)
+    probes_by_channel["all"] = transform_probes.transform(probes, node_data, False)
```

Comment:
Can we use something more descriptive than `True` & `False`?
Using a named argument or other ways.